metadata.yaml

```diff
@@ -13,8 +13,10 @@ description: |
   machines running on your infrastructure.
 maintainers:
   - Canonical Commercial Systems Team <jaas-crew@lists.canonical.com>
-website: https://ubuntu.com/security/livepatch
-docs: https://discourse.ubuntu.com/t/ubuntu-livepatch-service/22723
+website:
+  - https://ubuntu.com/security/livepatch
+  - https://discourse.ubuntu.com/t/ubuntu-livepatch-service/22723
+docs: https://discourse.charmhub.io/t/14446
```

Comment:
Why not link to the official tutorial?

Reply:
This should be a Charmhub Discourse post (on `discourse.charmhub.io`) so that it appears as the charm homepage on Charmhub.